internal/pkg/cli/deploy/svc.go

```diff
@@ -265,6 +244,16 @@ func newWorkloadDeployer(in *WorkloadDeployerInput) (*workloadDeployer, error) {
 	if err != nil {
 		return nil, fmt.Errorf("initiate env describer: %w", err)
 	}
+
```

Comment:
I guess the pros of the old way is to read and unmarshal the manifest only when it's needed (e.g. when the image wasn't able to build, it'd error out before we need the manifest - the read/unmarshal would be a waste). But this LGTM me too - less code, less function calls

Reply:
exactly. i removed this because i noticed recently added code use the cache var directly instead of calling that function, which is dangerous because if the func is never called before then they'll get the panic.